filters/builtin/compress.go

```diff
@@ -13,6 +12,8 @@ import (
 	"strings"
 	"sync"
 
+	"github.com/andybalholm/brotli"
```

Comment:
While not a big deal, this does not seem to be related to the refactoring.

Reply:
ah sorry, that's just my IDE sorting the imports